app/src/main/java/io/islnd/android/islnd/app/Dialogs.java

```diff
@@ -12,7 +12,8 @@
 
     public static void removeFriendDialog(Context context, int userId, String displayName) {
         AlertDialog.Builder builder = new AlertDialog.Builder(context, R.style.AppTheme_Dialog);
-        builder.setMessage("Remove " + displayName + " as a friend?")
+        builder.setTitle(context.getString(R.string.dialog_title_remove_friend))
+                .setMessage("Remove " + displayName + " as a friend?")
```

Comment:
We could store the string `"Remove %s as a friend?"` as a resource and create the message with `String.format`. Would make it a bit easier for translation if we ever got there....